Sources/PointFree/Episode/NewShow.swift

```diff
@@ -0,0 +1,45 @@
+import Either
+import HttpPipeline
+import Models
+import PointFreeRouter
+import Prelude
+import Tuple
+import Views
+
+let newEpisodeResponse: M<Tuple4<Either<String, Episode.Id>, User?, SubscriberState, Route?>>
+  = basicAuth(
+    user: Current.envVars.basicAuth.username,
+    password: Current.envVars.basicAuth.password
+    )
+    <<< fetchEpisodeForParam
+    <| writeStatus(.ok)
+    >=> userEpisodePermission
+    >=> map(lower)
+    >>> respond(
+      view: Views.newEpisodePageView(episodePageData:),
+      layoutData: { permission, episode, currentUser, subscriberState, currentRoute in
+        let previousEpisode = Current.episodes().first(where: { $0.sequence == episode.sequence - 1 })
+        let nextEpisode = Current.episodes().first(where: { $0.sequence == episode.sequence + 1 })
```

Comment:
I'm using the side bar to show only the previous/next episodes. I'm not integrating the collections stuff at all yet.